src/util.js

```diff
@@ -638,17 +638,18 @@ const util = {
    *                          or rejected with the Error of the last resolved Promise (if all promises are rejected)
    */
   anyPromise: function(promises) {
-    // eslint-disable-next-line no-async-promise-executor
-    return new Promise(async (resolve, reject) => {
-      let exception;
-      await Promise.all(promises.map(async promise => {
-        try {
-          resolve(await promise);
-        } catch (e) {
-          exception = e;
-        }
-      }));
-      reject(exception);
+    return new Promise((resolve, reject) => {
+      void (async () => {
+        let exception;
+        await Promise.all(promises.map(async promise => {
+          try {
+            resolve(await promise);
+          } catch (e) {
+            exception = e;
+          }
+        }));
+        reject(exception);
+      })();
```

Comment:
What's the point of this IIFE? Is it just to make the linter happy? The two versions are equivalent and the old one is shorter..

Reply:
The new version makes it clear the code is not "awaited"